primer/index.bs

```diff
@@ -710,37 +705,41 @@ Token Header:
         "alg": "EC",
         "crv": "P-256",
         "x": "N6VsICiPA1ciAA82Jhv7ykkPL9B0ippUjmla8Snr4HY",
-        "y": "ay9qDOrFGdGe_3hAivW5HnqHYdnYUkXJJevHOBU4z5s",
+        "y": "ay9qDOrFGdGe_3hAivW5HnqHYdnYUkXJJevHOBU4z5s"
     }
 }
 ```
-- `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` because we
+
+-   `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` because we
     generated the keys using eliptic curves.
-- `"typ": "dpop+jwt"`: The type of token. All DPoP Tokens should have a type of "dpop+jwt"
-- `"jwk": { "kty": "EC" ... }`: The client's public key.
+-   `"typ": "dpop+jwt"`: The type of token. All DPoP Tokens should have a type of "dpop+jwt"
+-   `"jwk": { "kty": "EC" ... }`: The client's public key.
 
 Token Body:
+
 ```json
 {
-    "htu": "https://auth.otherpod.example/token",
-    "htm": "POST",
-    "jti": "fb1264dd-fff1-4509-a7b1-0fe58d08d3e1",
-    "iat": 1603389616
+    "exp": 1603393216,
+    "htm": "get",
```

Comment:
Lower case is incorrect

Reply:
This is what it is in the encoded form above, so that needs to be fixed then.

Reply:
Separate issue then